controlled_resource_handler.go

```diff
@@ -96,11 +97,13 @@ func (g *ControlledResourceHandler) OnDelete(obj interface{}) {
 // This method may be called with an empty controllerName.
 func (g *ControlledResourceHandler) rebuildControllerByName(logger *zap.Logger, namespace, controllerName, addUpdateDelete string) {
 	if controllerName == "" {
+		logger.Info("Object has no controller, so nothing was enqueued")
 		return
 	}
 	logger.
-		With(logz.ControllerName(controllerName)).
```

Comment:
I guess we could just use "owner" as a term for our usage of ownerReference, since we only use if for parent-child relationship, and our object don't really have multiple "owners".